python/ecl/grid/ecl_grid.py

```diff
@@ -1316,6 +1349,16 @@ def export_position(self, index_frame):
         return data
 
     def export_corners(self, index_frame):
+        """Exports cell corner position coordinates to a numpy vector (matrix). 
+     
+        Index_fram must be a pandas dataframe with the same structure 
+        as obtained from export_index.  
+        In total there are eight 8 corners; and they are numbered as
+        described in elcl_grid.cpp L464.
```

Comment:
No - that line reference will go stale; please copy & paste the small illustration from the `.cpp` code in here.